make renode is broken

Running `make renode` in proj/proj_template results in this error:

```
make[1]: Leaving directory '/usr/local/google/home/avg/src/CFU-Playground/proj/proj_template/build'
/usr/local/google/home/avg/src/CFU-Playground/scripts/generate_renode_scripts.py /usr/local/google/home/avg/src/CFU-Playground/soc/build/digilent_arty.proj_template/csr.json digilent_arty /usr/local/google/home/avg/src/CFU-Playground/proj/proj_template/build/renode/ --repl /usr/local/google/home/avg/src/CFU-Playground/proj/proj_template/build/renode/digilent_arty_generated.repl
Traceback (most recent call last):
  File "/usr/local/google/home/avg/src/CFU-Playground/scripts/generate_renode_scripts.py", line 154, in <module>
    main()
  File "/usr/local/google/home/avg/src/CFU-Playground/scripts/generate_renode_scripts.py", line 139, in main
    f.write(generate_litex_renode_repl(args.conf_file, etherbone_peripherals, args.autoalign_memor_regions))
  File "/usr/local/google/home/avg/src/CFU-Playground/scripts/generate_renode_scripts.py", line 47, in generate_litex_renode_repl
    result = litex_renode.generate_repl(csr, etherbone_peripherals, autoalign)
TypeError: generate_repl() takes 2 positional arguments but 3 were given
make: *** [../proj.mk:176: renode-scripts] Error 1
```

I have seen this error in hps_accel, too.

--- a/proj/proj.mk
+++ b/proj/proj.mk
@@ -167,7 +167,6 @@ TARGET_REPL := $(BUILD_DIR)/renode/$(TARGET)_generated.repl
 .PHONY:	renode 
 renode: renode-scripts
 	@echo Running interactively under renode
-	$(COPY) $(SOFTWARE_ELF) $(PROJ_DIR)/renode/
 	pushd $(PROJ_DIR)/build/renode/ && $(RENODE_DIR)/renode -e "s @$(TARGET).resc" && popd
 
 .PHONY: renode-scripts

